src/Explorer/Controls/Settings/SettingsSubComponents/ConflictResolutionComponent.tsx

```diff
@@ -0,0 +1,150 @@
+import {
+  getTextFieldStyles,
+  conflictResolutionLwwTooltip,
+  conflictResolutionCustomToolTip,
+  subComponentStackProps,
+  titleAndInputStackProps
+} from "../SettingsRenderUtils";
+import { Label, Text, TextField, ITextFieldProps, Stack } from "office-ui-fabric-react";
+import { ToolTipLabelComponent } from "./ToolTipLabelComponent";
+
+export interface ConflictResolutionComponentProps {
+  collection: ViewModels.Collection;
+  container: Explorer;
+  conflictResolutionPolicyMode: StatefulValue<DataModels.ConflictResolutionMode>;
+  onConflictResolutionPolicyModeChange: (mode: DataModels.ConflictResolutionMode) => void;
+  conflictResolutionPolicyPath: StatefulValue<string>;
+  onConflictResolutionPolicyPathChange: (
+    event: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>,
+    newValue?: string
+  ) => void;
+  conflictResolutionPolicyProcedure: StatefulValue<string>;
+  onConflictResolutionPolicyProcedureChange: (
+    event: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>,
+    newValue?: string
+  ) => void;
+}
+
+export class ConflictResolutionComponent extends React.Component<ConflictResolutionComponentProps> {
+  constructor(props: ConflictResolutionComponentProps) {
+    super(props);
+  }
+
+  private onConflictResolutionCustomKeyPress = (event: React.KeyboardEvent<HTMLSpanElement>): void => {
+    if (event.charCode === Constants.KeyCodes.Space || event.charCode === Constants.KeyCodes.Enter) {
+      event.stopPropagation();
+      event.preventDefault();
+      this.onConflictResolutionCustomClick();
+    }
+  };
+
+  private onConflictResolutionCustomClick = (): void => {
+    this.props.onConflictResolutionPolicyModeChange(DataModels.ConflictResolutionMode.Custom);
+  };
+
+  private onConflictResolutionLWWKeyPress = (event: React.KeyboardEvent<HTMLSpanElement>): void => {
+    if (event.charCode === Constants.KeyCodes.Space || event.charCode === Constants.KeyCodes.Enter) {
+      event.stopPropagation();
+      event.preventDefault();
+      this.onConflictResolutionLWWClick();
+    }
+  };
+
+  private onConflictResolutionLWWClick = (): void => {
+    this.props.onConflictResolutionPolicyModeChange(DataModels.ConflictResolutionMode.LastWriterWins);
+  };
+
+  private getConflictResolutionModeComponent = (): JSX.Element => {
+    return (
+      <Stack {...titleAndInputStackProps}>
+        <Text>Mode</Text>
+        <div className="tabs" aria-label="Mode" role="radiogroup">
+          <div className="tab">
+            <Label
+              tabIndex={0}
+              role="radio"
+              className={`settingsV2Label ${this.props.conflictResolutionPolicyMode.isDirty() ? "dirty" : ""} ${
+                this.props.conflictResolutionPolicyMode.current === DataModels.ConflictResolutionMode.LastWriterWins
+                  ? "selectedRadio"
+                  : "unselectedRadio"
+              }`}
+              onKeyPress={this.onConflictResolutionLWWKeyPress}
```

Comment:
Can we use office-ui [ChoiceGroup](https://developer.microsoft.com/en-us/fluentui#/controls/web/choicegroup)? Or our [RadioSwitchComponent](https://github.com/Azure/cosmos-explorer/tree/master/src/Explorer/Controls/RadioSwitchComponent)? Or AccessibleElement. It would great if we can avoid creating another component. It is missing aria-label.